src/Entity/DoofinderApi.php

```diff
@@ -185,7 +185,7 @@ private function apiCall($entry_point = 'search', $params = [])
             return $response;
         }
 
-        if ($httpCode == 0) {
+        if (0 == $httpCode) {
```

Comment:
Strict comparisons are preferred over non-strict ones:

```suggestion
        if (0 === $httpCode) {
```

If you think that $httpCode might be an string, cast it to (int) :smile: